<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool</class>
 <widget class="QWidget" name="Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushDone">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSave">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDraw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>draw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushWrite">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushClear">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
